sdk/eventhubs/azure-messaging-eventhubs/pom.xml

```diff
@@ -42,7 +42,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-amqp</artifactId>
-      <version>1.3.0</version> <!-- {x-version-update;com.azure:azure-core-amqp;dependency} -->
+      <version>1.4.0</version> <!-- {x-version-update;unreleased_com.azure:azure-core-amqp;dependency} -->
```

Comment:
Was this done because the changes in AQMP Core breaks EventHubs?

Reply:
Just to make certain, we're not planning to release EventHubs from this branch right? Should release this an then update the trunk with the version increment PR.

Reply:
Yes, this was done only because the changes in AMQP core breaks EventHubs.  We will not be using this branch to release Event hubs.